Formula/kcli.rb

```diff
@@ -0,0 +1,112 @@
+class Kcli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Management tool for libvirt/aws/gcp/kubevirt/openstack/ovirt/vsphere/packet"
+  homepage "https://kcli.readthedocs.io/en/latest/"
+  url "https://github.com/karmab/kcli/archive/refs/tags/brew-v1.tar.gz"
+  sha256 "ee2f0842805a39d62d764cf7a71036062feaaae855d1c689e20023ab894d556e"
+  license "Apache-2.0"
+  head "https://github.com/karmab/kcli.git", branch: "master"
+
+  depends_on "libvirt"
+  depends_on "python@3.9"
```

Comment:
Can it work with 3.10?

Reply:
In fact, it doesn't https://github.com/Homebrew/homebrew-core/actions/runs/2608880913